src/main/java/com/selfxdsd/todocli/JsonTodosSerializer.java

```diff
@@ -99,12 +99,12 @@ public URI serialize() {
     private File getFile(final boolean deleteFirst) throws URISyntaxException,
         IOException {
         //folder where application is running.
-        final String parent = JsonTodosSerializer.class
+        final String parent = new File(JsonTodosSerializer.class
```

Comment:
@amihaiemil todos.json is now created in the same folder of the app jar (tested manually).